core/templates/dev/head/pages/exploration_player/AnswerClassificationService.js

```diff
@@ -107,31 +110,31 @@ oppia.factory('AnswerClassificationService', [
        * @param {function} interactionRulesService - The service which contains
        *   the explicit rules of that interaction.
        *
-       * @return {promise} A promise for an AnswerClassificationResult domain
-       *   object.
+       * @return {AnswerClassificationResult} The resulting
```

Comment:
Er, that's not true, right? The service is called an object factory because it creates AnswerClassificationResult objects, but the returned object is an AnswerClassificationResult domain object.